ew/cmd/slimeoid/slimeoidcmds.py

```diff
@@ -523,7 +526,11 @@ async def slimeoidbattle(cmd):
         challengee_slimeoid = EwSlimeoid(member=member)
 
         # Sets the bet to 0 if no bet is specified or if not at the arena.
-        bet = ewutils.getIntToken(tokens=cmd.tokens, allow_all=True)
```

Comment:
adding a lot of components onto the slimeoid battling options means they all have to get accessed via one command.

This gets complex, so if a !slimeoidbattle command is done without other tokens, it should display all possibilities in the response message. Something like:

"The syntax is !slimeoidbattle ([@player] /NPC trainer) bet [amount] size  [level]. A bet or level is optional."

Reply:
I don't think it fit too well into the responses. I added the suggested details to !commands slimeoids instead. I also added the new inventory filter terms to !commands items like i probably already should have. I could still try and put a response about the format into failed !slimeoidbattles though.